modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,969 @@
+|`ami-039021677b936e7d1`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-06f9ed4481d613bc6`
+|`rhcos-45.81.202005200134-0`
+|`ami-028d06116fead2ccf`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-02c5c09b4203097b2`
+|`rhcos-46.82.202008071940-0`
+|`ami-05c9105cc61320b72`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-06191e0b43a6bfacf`
+|`rhcos-46.82.202008181646-0`
+
+.19+|*eu-west-3*
+|`ami-068d3dd3c638bd157`
+|`rhcos-46.82.202008071940-0`
+|`ami-0f5ad1dbc8722f7dc`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-0c3a1026ac3f99cca`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-06d868044e94b4c3d`
+|`rhcos-48.84.202107202156-0`
+|`ami-0d5f2c4aaf3302d7e`
+|`rhcos-45.81.202005200134-0`
+|`ami-0eee03b7f795ca880`
+|`rhcos-48.84.202105062123-0`
+|`ami-0b17362653f2397bf`
+|`rhcos-49.84.202107191624-0`
+|`ami-0f581d9ec4d7db141`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-075ebd9414a260c34`
+|`rhcos-46.82.202008181646-0`
+|`ami-05967a4e65a8356d9`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-04d292a1e2821f359`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-0cc15eae9e71c3d26`
+|`rhcos-47.83.202102090044-0`
+|`ami-0e6a6a923427e76a6`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0d1987bdab0cc7903`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-063c6f9a727602a3b`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-0e2a55691789771d0`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-09b8632b92e5edac5`
+|`rhcos-45.82.202008010929-0`
+|`ami-074c99ae8530e4eb3`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-02c056bb061ecada0`
+|`rhcos-415.92.202309201557-0-x86_64`
+
+.6+|*eu-south-2*
+|`ami-0e7c6506841f137c0`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0b8953c88a0645080`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-031e602fbf1087ed3`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-06838fa4e8b987cd8`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0f3ad9b25036a95dc`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-0880fa6c19e3d426a`
+|`rhcos-411.85.202203242008-0-x86_64`
+
+.19+|*eu-north-1*
+|`ami-09c872e8081e9d63d`
+|`rhcos-45.81.202005200134-0`
+|`ami-080f460264d851577`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-0f8142237d51621db`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-099f3930c1ec7cc46`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-02fbec4b51b9c4793`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0435b2118cccddb6b`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-076637160d6b221ff`
+|`rhcos-45.82.202008010929-0`
+|`ami-07d141e3e87a9dc80`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0e8f22283e46e8cb3`
+|`rhcos-46.82.202008181646-0`
+|`ami-00be34c2341c4d4e2`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-04bc6ffc11b85dcc9`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-0e84383413e10ca0c`
+|`rhcos-47.83.202102090044-0`
+|`ami-0310e8f4a9fe48b28`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0bbdaf5e44a35ae8e`
+|`rhcos-49.84.202107191624-0`
+|`ami-0194bfc5560eb87b3`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
```

Comment:
Can we clarify why the `[Copied ami-.... from us-east-1]` is there? Can we remove it?